src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/tests/latest/test_ams_asset_filter_scenarios.py

```diff
@@ -0,0 +1,266 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+
+from azure.cli.core.util import CLIError
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer, StorageAccountPreparer
+
+
+class AmsAssetFilterTests(ScenarioTest):
+    def _get_test_data_file(self, filename):
+        filepath = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data', filename)
+        self.assertTrue(os.path.isfile(filepath), 'File {} does not exist.'.format(filepath))
+        return filepath
+
+    @ResourceGroupPreparer()
+    @StorageAccountPreparer(parameter_name='storage_account_for_create')
+    def test_ams_asset_filter_create(self, storage_account_for_create):
+        amsname = self.create_random_name(prefix='ams', length=12)
+        container = self.create_random_name(prefix='cont', length=8)
+
+        self.kwargs.update({
+            'amsname': amsname,
+            'storageAccount': storage_account_for_create,
+            'location': 'westus2',
+            'container': container
+        })
+
+        self.cmd('az ams account create -n {amsname} -g {rg} --storage-account {storageAccount} -l {location}')
+        self.cmd('az ams account storage add -a {amsname} -g {rg} -n {storageAccount}')
+
+        assetName = self.create_random_name(prefix='asset', length=12)
```

Comment:
You should use snake_case in python variables and camelCase for kwargs

Reply:
agree